runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -283,20 +284,33 @@ jvmtiGetFrameCount(jvmtiEnv* env,
 		rc = getVMThread(currentThread, thread, &targetThread, TRUE, TRUE);
 		if (rc == JVMTI_ERROR_NONE) {
 			J9StackWalkState walkState;
-
-			vm->internalVMFunctions->haltThreadForInspection(currentThread, targetThread);
-
-			walkState.walkThread = targetThread;
 			walkState.flags = J9_STACKWALK_INCLUDE_NATIVES | J9_STACKWALK_VISIBLE_ONLY;
 			walkState.skipCount = 0;
-			vm->walkStackFrames(currentThread, &walkState);
+
+#if JAVA_SPEC_VERSION >= 19
+			if (NULL == targetThread) {
+				j9object_t threadObject = J9_JNI_UNWRAP_REFERENCE(thread);
+				if (IS_VIRTUAL_THREAD(currentThread, threadObject)) {
+					j9object_t contObject = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, threadObject);
+					internalVMFunctions->walkContinuationStackFrames(currentThread, contObject, &walkState);
+				} else {
+					Assert_JVMTI_true(0);
```

Comment:
you can add a new `Assert_JVMTI_unreachable` to j9jvmti.tdf

Reply:
Since this is unreachable, why can't we just skip the `IS_VIRTUAL_THREAD` check because it's always true?